$OpenBSD: patch-backend_ipp_c,v 1.1 2012/02/16 22:12:08 ajacoutot Exp $

Increase timeout so that it doesn't fail on low-end configurations.

--- backend/ipp.c.orig	Mon Feb  6 11:17:34 2012
+++ backend/ipp.c	Mon Feb  6 11:17:43 2012
@@ -634,7 +634,7 @@ main(int  argc,				/* I - Number of command-line args 
   }
 
   http = _httpCreate(hostname, port, addrlist, cupsEncryption(), AF_UNSPEC);
-  httpSetTimeout(http, 30.0, timeout_cb, NULL);
+  httpSetTimeout(http, 90.0, timeout_cb, NULL);
 
  /*
   * See if the printer supports SNMP...
@@ -1980,7 +1980,7 @@ monitor_printer(
 
   http = _httpCreate(monitor->hostname, monitor->port, NULL, monitor->encryption,
                      AF_UNSPEC);
-  httpSetTimeout(http, 30.0, timeout_cb, NULL);
+  httpSetTimeout(http, 90.0, timeout_cb, NULL);
   cupsSetPasswordCB(password_cb);
 
  /*
